src/tree/mod.rs

```diff
@@ -1,6 +1,6 @@
 use std::io;
 
-use dusk_bls12_381::Scalar;
+use dusk_plonk::bls12_381::Scalar;
```

Comment:
We renamed everywhere else to `BlsScalar` but not here. Why?

Reply:
Because I missed it hahah. Good catch!